src/zero_copy/error.rs

```diff
@@ -52,9 +52,9 @@ use super::*;
 ///     None => Err(MyError::NotADigit(span, c)),
 /// });
 ///
-/// assert_eq!(numeral.parse("3").0, Some(3));
-/// assert_eq!(numeral.parse("7").0, Some(7));
-/// assert_eq!(numeral.parse("f").1, vec![MyError::NotADigit(0..1, 'f')]);
+/// assert_eq!(numeral.parse("3"), ParseResult::Ok(3));
```

Comment:
I think it's probably worth using `.parse("3").into_result()` and then comparing with `Ok(...)` in the examples to minimise confusion.